galley/pkg/config/analysis/diag/level.go

```diff
@@ -15,15 +15,22 @@
 package diag
 
 // Level is the severity level of a message.
-type Level string
+type Level struct {
+	severity int
```

Comment:
nit: since we're only using this for sort order, we should either name or comment accordingly. We don't want anything else using this number, since if we add more levels (e.g. "Fatal" or equivalent) it could change. 

Alternately, we could make this a [proper enum](https://yourbasic.org/golang/iota/#complete-enum-type-with-strings-best-practice), but that's probably overkill.